Domains/4-Application/Connectors/ConceptStation/CSPhysical.ecschema.xml

```diff
@@ -15,75 +15,53 @@
     </ECEntityClass>
-    <ECEntityClass typeName="HyperloopSegment" modifier="Sealed" displayLabel="Hyperloop Segment" description="A physical stretch of hyperloop that groups all components across its width." >
+    <ECEntityClass typeName="HyperloopSegment" modifier="Sealed" displayLabel="Hyperloop Segment" description="A physical stretch of hyperloop that groups all components across its width.">
         <BaseClass>SegmentAssembly</BaseClass>
     </ECEntityClass>
     <ECEntityClass typeName="Furniture" modifier="Sealed" displayLabel="Furniture" description="A physical piece of furniture.">
-        <BaseClass>SegmentAssembly</BaseClass>
-        <ECProperty propertyName="StartStation" typeName="double">
-            <ECCustomAttributes>
-                <HiddenProperty xmlns="CoreCustomAttributes.01.00.03"/>
-            </ECCustomAttributes>
-        </ECProperty>
-        <ECProperty propertyName="EndStation" typeName="double">
-            <ECCustomAttributes>
-                <HiddenProperty xmlns="CoreCustomAttributes.01.00.03"/>
-            </ECCustomAttributes>
-        </ECProperty>
+        <BaseClass>ConceptStationPhysicalElement</BaseClass>
     </ECEntityClass>
     <ECEntityClass typeName="Culvert" modifier="Sealed" displayLabel="Culvert" description="A physical culvert structure.">
-        <BaseClass>SegmentAssembly</BaseClass>
-        <ECProperty propertyName="StartStation" typeName="double">
-            <ECCustomAttributes>
-                <HiddenProperty xmlns="CoreCustomAttributes.01.00.03"/>
-            </ECCustomAttributes>
-        </ECProperty>
-        <ECProperty propertyName="EndStation" typeName="double">
-            <ECCustomAttributes>
-                <HiddenProperty xmlns="CoreCustomAttributes.01.00.03"/>
-            </ECCustomAttributes>
-        </ECProperty>
-    </ECEntityClass>
-    <ECEntityClass typeName="CADModeling" modifier="Sealed" displayLabel="CAD Modeling" description="A physical structure modeling with CAD shapes.">
-        <BaseClass>SegmentAssembly</BaseClass>
-        <ECProperty propertyName="StartStation" typeName="double">
-            <ECCustomAttributes>
-                <HiddenProperty xmlns="CoreCustomAttributes.01.00.03"/>
-            </ECCustomAttributes>
-        </ECProperty>
-        <ECProperty propertyName="EndStation" typeName="double">
-            <ECCustomAttributes>
-                <HiddenProperty xmlns="CoreCustomAttributes.01.00.03"/>
-            </ECCustomAttributes>
-        </ECProperty>
+        <BaseClass>ConceptStationPhysicalElement</BaseClass>
+    </ECEntityClass>
+    <ECEntityClass typeName="SmartLine" modifier="Sealed" displayLabel="CAD Modeling line element" description="An opened line or closed shape.">
+        <BaseClass>ConceptStationPhysicalElement</BaseClass>
+    </ECEntityClass>
+    <ECEntityClass typeName="ExtrudedMesh" modifier="Sealed" displayLabel="CAD Modeling mesh element" description="An extruded smart line.">
```

Comment:
I think these CAD-related classes need a clearer description. What about "A mesh element imported into ConceptStation from a CAD source."

Reply:
Extruded mesh are also created in ConceptStation.

Maybe "A mesh element created by the extrusion of a SmartLine."